manifests/Tencent/QQ/0.0.0.0.yaml

```diff
@@ -0,0 +1,19 @@
+Id: Tencent.QQ
+Name: QQ
+Version: 0.0.0.0
+Publisher: hifocus
```

Comment:
```suggestion
Publisher: Tencent Corporation
```

Reply:
see `https://github.com/microsoft/winget-pkgs/pull/564`

Reply:
> 

See https://apps.apple.com/us/app/qq/id444934666

I would think it's a better idea to use `Tencent Technology (Shenzhen) Company Limited`.